stix_edh/bindings/cyber_profile.py

```diff
@@ -442,7 +444,6 @@ def exportChildren(self, lwrite, level, nsmap, namespace_=XML_NS, name_='Declass
             showIndent(lwrite, level, pretty_print)
             lwrite('<%s:declassEvent>%s</%s:declassEvent>%s' % (nsmap[namespace_], quote_xml(self.declassEvent), nsmap[namespace_], eol_))
     def build(self, node):
-        self.__sourcenode__ = node
```

Comment:
Is this related to sharing scope? All the other `build()` functions include this line so I'm nervous about removing it here.

Reply:
Same - didn't bring it over